src/compiler/crystal/syntax/parser.cr

```diff
@@ -333,6 +332,11 @@ module Crystal
               raise "can't change the value of self", location
             end
 
+            if atomic.is_a?(Call) && (atomic.name.ends_with?('?') ||
```

Comment:
You need to remove the part from this line. Couldn't post a multi-line diff suggestion.